nomossa build failure

<!-- Before filling this issue, please read the wiki (https://github.com/fossology/fossology/wiki)
and search if the bug do not already exists in the issues (https://github.com/fossology/fossology/issues). -->

### Description

With a built `lib` (running `make` at top level), the `nomossa` build fails.

#### How to reproduce

`make` top level directory, build `nomossa` (`make -C src/nomos/agent -f Makefile.sa all`)

### Versions

* Last commit id on master: 7cd0d95874406a6801977ae57e01293419020cd1
* Operating System (lsb_release -a):
```
Distributor ID:	Debian
Description:	Debian GNU/Linux 10 (buster)
Release:	10
Codename:	buster
```

### Logs

```
make: Entering directory 'src/nomos/agent'
gcc nomos.o standalone.o licenses.o list.o parse.o process.o nomos_regex.o util.o nomos_gap.o nomos_utils.o doctorBuffer_utils.o json_writer.o  _precheck.o _autodata.o -DSTANDALONE -Wall -D_FILE_OFFSET_BITS=64 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/json-c -lfossology -L../../../src/lib/c -lglib-2.0 -lpq  -lglib-2.0 -ljson-c -lpthread -lrt -o nomossa
/usr/bin/ld: ../../../src/lib/c/libfossology.a(libfossagent.o): in function `fo_GetAgentKey':
src/lib/c/libfossagent.c:173: multiple definition of `fo_GetAgentKey'; standalone.o:standalone.c:(.text+0x76): first defined here
/usr/bin/ld: ../../../src/lib/c/libfossology.a(libfossagent.o): in function `fo_WriteARS':
src/lib/c/libfossagent.c:230: multiple definition of `fo_WriteARS'; standalone.o:standalone.c:(.text+0x95): first defined here
/usr/bin/ld: ../../../src/lib/c/libfossology.a(libfossagent.o): in function `GetUploadPerm':
src/lib/c/libfossagent.c:386: multiple definition of `GetUploadPerm'; standalone.o:standalone.c:(.text+0x12c): first defined here
collect2: error: ld returned 1 exit status
make: *** [Makefile.sa:31: nomossa] Error 1
make: Leaving directory 'src/nomos/agent'
```

--- a/src/nomos/agent/standalone.c
+++ b/src/nomos/agent/standalone.c
@@ -30,6 +30,9 @@ void fo_dbManager_free(fo_dbManager* dbManager) {}
 fo_dbManager_PreparedStatement* fo_dbManager_PrepareStamement_str(fo_dbManager* dbManager, const char* name, const char* query, const char* paramtypes) {return NULL;}
 PGresult* fo_dbManager_ExecPrepared(fo_dbManager_PreparedStatement* preparedStatement, ...) {return NULL;}
 
+PGresult* getSelectedPFiles(PGconn* pgConn, int uploadPk, int agentPk, bool ignoreFilesWithMimeType) {return NULL;}
+PGresult* checkDuplicateReq(PGconn* pgConn, int uploadPk, int agentPk) {return NULL;}
+
 //ExecStatusType PQresultStatus(const PGresult *res);
 int PQresultStatus(const PGresult *res){ return(PGRES_COMMAND_OK);}
 char *PQresultErrorMessage(const PGresult *res){return(0);}

--- a/src/nomos/agent/standalone.h
+++ b/src/nomos/agent/standalone.h
@@ -1,6 +1,7 @@
 #ifndef _STANDALONE_H
 #define _STANDALONE_H 1
 #include <stdlib.h>
+#include <stdbool.h>
 
 typedef int PGconn;
 typedef enum
@@ -88,6 +89,10 @@ fo_dbManager* fo_dbManager_new(PGconn* dbConnection);
 void fo_dbManager_free(fo_dbManager* dbManager);
 fo_dbManager_PreparedStatement* fo_dbManager_PrepareStamement_str(fo_dbManager* dbManager, const char* name, const char* query, const char* paramtypes);
 PGresult* fo_dbManager_ExecPrepared(fo_dbManager_PreparedStatement* preparedStatement, ...);
+
+PGresult* getSelectedPFiles(PGconn* pgConn, int uploadPk, int agentPk, bool ignoreFilesWithMimeType);
+PGresult* checkDuplicateReq(PGconn* pgConn, int upload_pk, int agentPk);
+
 
 //ExecStatusType PQresultStatus(const PGresult *res);
 extern int PQresultStatus(const PGresult *res);

